tripy/tripy/config.py

```diff
@@ -38,6 +38,8 @@
 enable_tensorrt_debug = os.environ.get("TRIPY_TRT_DEBUG_ENABLED", "0") == "1"
 tensorrt_debug_path = os.environ.get("TRIPY_TRT_DEBUG_PATH", os.path.join("/", "tripy", "tensorrt-dumps"))
 
+eager_cache = os.environ.get("TRIPY_EAGER_CACHE", "0") == "1"
```

Comment:
Will need to export this in the public API and document it before merging (similar to `timing_cache_file_path`)

Reply:
should the public doc be in a new page? or under `debugging.md`?

Reply:
`debugging.md` is probably fine to start with

Reply:
okay sounds good